packages/elements/src/components/ino-datepicker/ino-datepicker.tsx

```diff
@@ -244,6 +244,26 @@ export class Datepicker implements ComponentInterface {
     this.flatpickr.toggle();
   }
 
+  /**
+   * Displays the datepicker as invalid if set to true.
+   * If the property is not set or set to false,
+   * the validation is handled by the default validation.
```

Comment:
Shouldn't the error prop have three states:
* `null|undefined` => use native validation
* `true` => has error
* `false` => has no error

If I integrate the component and manually set the error=false, I wouldn't suggest that the component still natively validates.

This may be something we should unify across all components